test/test_projects/c.py

```diff
@@ -17,7 +17,7 @@
         return NULL;
 
     // Spam should not be allowed through the filter.
-    sts = strcmp(content, "spam");
+    sts = strcmp(content, "spam") != 0;
```

Comment:
I'm curious why this change?

Reply:
It makes the return value predictable, so it can be used in tests. [`strcmp`](https://www.man7.org/linux/man-pages/man3/strcmp.3.html) only promises to return a negative, positive or zero value; it doesn't specify what the negative or positive values are.